src/ua.c

```diff
@@ -460,6 +462,7 @@ static void call_event_handler(struct call *call, enum call_event ev,
 			break;
 
 		case ANSWERMODE_MANUAL:
+		default:
```

Comment:
why is this needed ? answer module has 3 values that are all handled in this switch:

```c
enum answermode {
	ANSWERMODE_MANUAL = 0,
	ANSWERMODE_EARLY,
	ANSWERMODE_AUTO
};
```